API.md

```diff
@@ -67,7 +67,8 @@ of iD (e.g. `https://ideditor-release.netlify.app`), the following parameters ar
 
 When constructing a URL to an instance of iD embedded in the OpenStreetMap Rails
 Port (e.g. `http://www.openstreetmap.org/edit?editor=id`), the following parameters
-are available as **regular URL query parameters**:
+are available as **URL hash-query parameters, to put after a # instead of a ?**.  
+_Example:_ `https://www.openstreetmap.org/edit#gpx=https://tasks.hotosm.org/project/592/task/16.gpx`
```

Comment:
Unfortunately, the link to that GPX file doesn't seem to work here.

Reply:
Yes, but same link is mentioned in the same page above also. I assumed keeping the same here will give a consistent logical flow to help someone reading the documentation to follow properly. I do have valid GPX links but it'll end up exposing my server to more bots etc. Let me try hosting one in a safe place.

Reply:
ok how about this: https://www.openstreetmap.org/edit#gpx=https://gist.githubusercontent.com/answerquest/9445352b60ca5b44714675eae00f243a/raw/56a6343a29223318f4a697bfd16cbb2c3b8155ad/sample_boundary.gpx

Reply:
updated. pls tell if you need me to change the gpx link in top portion of the page too in the standalone iD section. I'll prefer to leave that be.

Reply:
Example works fine now, thanks!